src/components/footer/Footer.tsx

```diff
@@ -199,7 +199,7 @@ const AppFooter = () => {
         companyLink={pagoPALink}
         postLoginLinks={postLoginLinks}
         preLoginLinks={preLoginLinks}
-        currentLangCode={i18n.language as LangCode}
+        currentLangCode={i18n.languages ? (i18n.languages[0] as LangCode) : 'it'}
```

Comment:
Solo una curiosità: perchè qui si è preferito accedere alla linga corrente con `i18n.languages[0]` invece di  `i18n.language`?

Reply:
Il language detector di i18n trova due languageCode, quello con il local(en-uk), e quello senza(en). Avendo a disposizione due languageCode, mentre a livello applicativo avviene in automatico il fallback sul codice senza local, in quanto è l'unico implementato, nella visualizzazione del linguaggio selezionato nel footer viene selezionato tramite i18n.languages[0] per essere sicuri che nella prima visualizzazione ci sia il code senza local(il quale è l'unico con assegnata una label , in quanto unico implementato), mentre nelle successive visualizzazioni verrà visualizzato il code presente nei cookie, oppure quello selezionato precedentemente dall'utente in caso quest'ultimo abbia cambiato la scelta. Usassimo quindi i18n.language per la visualizzazione nel footer, nella prima visualizzazione non avremmo nulla perchè i18n.language è di default con local(en-uk).